src/aiortc/rtcrtpsender.py

```diff
@@ -170,6 +174,12 @@ def replaceTrack(self, track: Optional[MediaStreamTrack]) -> None:
     def setTransport(self, transport) -> None:
         self.__transport = transport
 
+    def setPlayoutDelay(self, min, max) -> None:
+        mask = 0xFFF
+        self.__playout_delay = ((min & mask) << 12) | (max & mask)
+        self.__playout_delay_first_written = float("inf")
```

Comment:
Is this member a float or an int?

Reply:
Both, so that comparison with sequence number always pass, but I was not aware it had to deal with uint16.
So I think this won't work, I'll change it.